recipe/meta.yaml

```diff
@@ -7,8 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: graphviz-{{ version }}.zip
-  url: https://files.pythonhosted.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
+  fn: python-graphviz-{{ version }}.zip
```

Comment:
As mentioned in #9, I would rather prefer to keep the official name for the sdist download